portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +14,46 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.List;
+import java.util.LinkedList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments through post and TODO(rossjohnson): get requests */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+    public static final String JSON_CONTENT_TYPE = "application/json";
+    public static final String COMMENT_PROPERTY = "comment";
+    public static final String TIMESTAMP_PROPERTY = "timestamp";
+    public static final String COMMENT_ENTITY = "comment";
+    private static final String HOME_HTML = "/index.html";
+    private final List<String> greetingsList = new LinkedList<String>();
+    private final List<String> commentList = new LinkedList<String>();
+    private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+      //TODO(rossjohnson): Implement a function that retrieves comments from the database and returns them to the user
   }
```

Comment:
indentation, generally it is two space indentation instead of tab / 4 space indentation at Google.